src/main/webapp/index.jsp

```diff
@@ -0,0 +1,234 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
+    <title>NeighborHelp</title>
+    <link rel="stylesheet" href="homepage_style.css">
+    <script src="homepage_script.js"></script>
+    <script src="https://kit.fontawesome.com/71105f4105.js" crossorigin="anonymous"></script> 
+  </head>
+  <body>
+      <!--Site Header-->
+      <header>
+          <nav>
+              <div id="userpage-icon">
+                  <a href="user_profile.html">
+                      <i class="fas fa-user-circle fa-3x" title="Go to User Page"></i>
+                  </a>
+              </div>
+          </nav>
+          <div id="title">
+              NeighborHelp
+          </div>
+      </header>
+
+      <!--Main Content of Site-->
+      <section>
+
+          <!--Control Bar for choosing categories and adding tasks-->
+          <div id="control-bar">
+              <div id="categories">
+                  <div class="categories" id="category-all" onclick="filterBy('all')">ALL</div>
+                  <div class="categories" id="category-garden" onclick="filterBy('garden')">GARDEN</div>
+                  <div class="categories" id="category-shopping" onclick="filterBy('shopping')">SHOPPING</div>
+                  <div class="categories" id="category-pets" onclick="filterBy('pets')">PETS</div>
+                  <div class="categories" id="category-misc" onclick="filterBy('misc')">MISC</div>
+              </div>
+              <div id="add-task">
+                  <i class="fas fa-plus-circle" aria-hidden="true" id="addtaskbutton" title="Add Task" onclick="showModal()"></i>
+              </div>
+          </div>
+
+          <!--Hard Coded List of Tasks-->
+          <div id="tasks-list">
```

Comment:
For the user page, I used `<table>` to list all the tasks. We can discuss about whether we want to use `<table>` for tasks or use `<div>` instead.

Reply:
I was thinking about that too. I like using `<div>`s since it gives me more flexibility when aligning things, but we also might not need to display the tasks in the same way on both pages, since on the main page you only see the basic details, and on the users page it's more of an progress status display. I don't know haha, let's see what others say :)